tox.ini

```diff
@@ -25,32 +23,41 @@ extras =
 [testenv:lint]
 basepython = python3
 skip_install = true
-deps =
-    flake8
-    check-manifest
-    check-python-versions
-    wheel
 commands =
+    isort --check-only --diff {toxinidir}/src {toxinidir}/setup.py
     flake8 src setup.py
     check-manifest
     check-python-versions
+deps =
+    check-manifest
+    check-python-versions >= 0.19.1
+    wheel
+    flake8
+    isort
+
+[testenv:isort-apply]
+basepython = python3
+skip_install = true
+commands_pre =
+deps =
+    isort
+commands =
+    isort {toxinidir}/src {toxinidir}/setup.py []
```

Comment:
I only noticed this now, but this `[]` seems broken to me...

```
$ isort *.py []
Broken 1 paths
```

`isort` still does its job, but then it reports this `Borken 1 paths`.

Reply:
`tox` replaces `[]` by the command line arguments `tox` was called with. So calling `tox -eisort-apply -- -v` actually calls `isort /.../z3c.menu.ready2go/src /.../z3c.menu.ready2go/setup.py -v`.